tests/Hedgehog.Tests/GenTests.fs

```diff
@@ -30,39 +42,49 @@ let genTests = testList "Gen tests" [
         let ticks =
             Random.integral range
             |> Random.run seed1 0
-        let expected = System.DateTime ticks
 
-        let actual = Gen.dateTime (Range.constant System.DateTime.MinValue System.DateTime.MaxValue)
+        let actual =
+            (Range.constant System.DateTime.MinValue System.DateTime.MaxValue)
+            |> Gen.dateTime
+            |> Gen.toRandom
+            |> Random.run seed0 0
+            |> Tree.outcome
 
-        let result = actual |> Gen.toRandom |> Random.run seed0 0 |> Tree.outcome
-        expected =! result
+        let expected =
+            System.DateTime ticks
+        expected =! actual
 
     testCase "dateTime shrinks to correct mid-value" <| fun _ ->
-        let result =
+        let actual =
             property {
                 let! actual =
-                  Range.constantFrom (System.DateTime (2000, 1, 1)) System.DateTime.MinValue System.DateTime.MaxValue
+                  (Range.constantFrom
+                       (System.DateTime (2000, 1, 1))
+                        System.DateTime.MinValue
+                        System.DateTime.MaxValue)
                   |> Gen.dateTime
                 System.DateTime.Now =! actual
             }
             |> Property.report
             |> Report.render
-            |> (fun x -> x.Split ([|System.Environment.NewLine|], System.StringSplitOptions.None))
+            |> (fun x -> x.Split [| '\n' |])
```

Comment:
Do we want to always assume `\n` here? Does this break Windows users?

Reply:
No, I made a mistake here. I'll revert that one. Very good catch :+1:

Reply:
Done.